module4/projects/mod4Project/inception.md

```diff
@@ -0,0 +1,114 @@
+    * Eliminate duplicate ideas
+* **Instructions**
+  * You will be organizing these thoughts as a group in the project's Miro board
+  * Each person should present their features one at a time, giving a *brief* description of what that feature is, so it is clear for the whole group.
+  * After you have described the feature move the sticky note into a category. If it doesn't fit in an existing category, make one.
+  * As each person takes a turn you will notice that some of the features will be duplicates of previously shared features. If this is the case, simple delete that sticky note. _Don't spend time describing a feature if it is a duplicate._
+  * Come up with a name for each of the categories. These will be referred to as [EPICs](https://www.yodiz.com/blog/what-is-epic-in-agile-methodology-definition-and-template-of-epic/) and will be a way to organize your features in your project management tool.
+
+### Feature Prioritization and MVP Definition (Decide)
+
+* **Objectives**
+    * Define the features that will make up your [**Minimum Viable Product**](https://www.agilealliance.org/glossary/mvp/#q=~(infinite~false~filters~(tags~(~'mvp))~searchTerm~'~sort~false~sortDirection~'asc~page~1))
+* **Instructions (recommended)**
+    * Use labels/colors in your miro board to determine MVP/extension/unsure stories.
+    * If you are unsure about a feature, table the feature and come back to it at the end.
+    * Repeat this process until you have gone through all of the stories.
+    * Once you are finished, you have your MVP definition.
+
+
+### Wireframing (Solve)
+
+* **Objectives**
+    * Decide how users will interact with your application
+* **Instructions (required)**
+    * Create a high level [user flow diagram](https://bashooka.com/inspiration/33-excellent-user-flow-examples-for-inspiration/) that lists the steps a user will take when interacting with your application.
+    * Make a list of all of the pages required for your application along with what features will exist on each page.
+    * For each page, create a [wireframe](https://www.usability.gov/how-to-and-tools/methods/wireframing.html) by sketching out what that user will see when they navigate to that page. Include details such as header text, buttons, drop downs, forms with the specific fields, graphs etc.
+    * Do a few pages together as a group, then divide and conquer the remaining pages.
+    * [Figma](https://www.figma.com/wireframe-tool/?utm_source=google&utm_medium=cpc&utm_campaign=17340391121&utm_term=free%20wireframe%20tools&utm_content=600487515776&gclid=CjwKCAjw5P2aBhAlEiwAAdY7dLtWlIVK0NWWHUXbjj8KMAaBN8P1lO16khOpesJ63utNI5b-cF1e2xoClmoQAvD_BwE) is a great tool for wireframing
+
+✅Deliverable: Send a link to your project manager with your wireframes.
+
+### Story Writing (Solve)
+
+* **Objectives**
+    * Document technical requirements for the development team
+* **Instructions (required)**
+    * Create a project board using Github Projects or Trello.
```

Comment:
I think we decided to remove trello as an option here and say Github Projects is the way.